soul-client/soul-client-http/soul-client-springmvc/src/main/java/org/dromara/soul/client/springmvc/init/SpringMvcClientBeanPostProcessor.java

```diff
@@ -74,9 +73,8 @@ public Object postProcessAfterInitialization(@NonNull final Object bean, @NonNul
             return bean;
         }
         Controller controller = AnnotationUtils.findAnnotation(bean.getClass(), Controller.class);
-        RestController restController = AnnotationUtils.findAnnotation(bean.getClass(), RestController.class);
```

Comment:
after， also support RestController  Annotation？

Reply:
Yes, it does.  It will work like this:
![image-20210202135138034](https://gitee.com/stephenshen/pic-bed/raw/master/img/20210202135138.png)